frontend/.env

```diff
@@ -2,7 +2,7 @@ REACT_APP_TITLE=SimpleReport
 REACT_APP_DESCRIPTION=SimpleReport Test Management Application
 REACT_APP_ICON=usds-32x32.png
 REACT_APP_BACKEND_URL=
-REACT_APP_OKTA_URL=https://simplereport.gov/
+REACT_APP_BASE_URL=https://simplereport.gov/
```

Comment:
Is there any instance in which we'd want the base url and backend url set generally for all environments? I think we'd rather it fail than use these default values in the wrong environment.

I think this is probably also true for the app insights key, but would require adding it individually to each environment

Reply:
Probably not. We can update the `BACKEND_URL` to be the `BACKEND_PATH` setting it to null locally and `/api` in our cloud envs however I would rather us fix our infrastructure to not serve from `/app` and `/api` as they cause issues where the direct link works fine when I am testing locally and then breaks in dev/prod